drivers/sensor/bosch/bme280/bme280.h

```diff
@@ -206,6 +206,14 @@ struct bme280_decoder_header {
 
 struct bme280_encoded_data {
 	struct bme280_decoder_header header;
+	struct {
```

Comment:
This seems like it should be in the decoder header

Reply:
I used BMA4xx driver as a "guide" and tried to be consistent.

```
struct bma4xx_encoded_data {
	struct bma4xx_decoder_header header;
	struct {
		/** Set if `accel_xyz` has data */
		uint8_t has_accel: 1;
		/** Set if `temp` has data */
		uint8_t has_temp: 1;
		uint8_t reserved: 6;
	} __attribute__((__packed__));
	int16_t accel_xyz[3];
#ifdef CONFIG_BMA4XX_TEMPERATURE
	int8_t temp;
#endif /* CONFIG_BMA4XX_TEMPERATURE */
};
```

Reply:
Fair enough!